store/download_test.go

```diff
@@ -149,7 +155,7 @@ func (s *downloadSuite) TestDownloadCancellation(c *C) {
 		io.WriteString(w, "foo")
 		syncCh <- struct{}{}
 		io.WriteString(w, "bar")
-		time.Sleep(time.Duration(1) * time.Second)
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Won't this fail occasionally on a slow machine? Maybe we could make this test conditional on `testing.Short()`?

Reply:
it shouldn't, but it might. If it does we can add the conditional and drop the timeouts back, ok?

Reply:
Yes, that would be fair. @mvo5 wdyt?